pac4j-jwt/src/main/java/org/pac4j/jwt/credentials/authenticator/JwtAuthenticator.java

```diff
@@ -70,7 +84,17 @@ private void warning() {
      * @since 1.8.2
      */
     public JwtAuthenticator(final String signingSecret, final String encryptionSecret) {
-        this.signingSecret = signingSecret;
+    	setSigningSecret(signingSecret);
```

Comment:
Certainly, but we must help pac4j adopters to use the library: so there must be somewhere (in the Javadoc header class or in this method Javadoc) a explanation and example. So we can use simple strings as we use to but also the value of a certificate, right?